client/pages/artist/_itemId/index.vue

```diff
@@ -3,13 +3,20 @@
     <template #left>
       <v-row justify="center" justify-sm="start">
         <v-col cols="6" sm="3" class="d-flex flex-row">
-          <v-img
-            v-if="item.ImageTags && item.ImageTags.Primary"
-            class="person-image elevation-2"
-            cover
-            aspect-ratio="1"
-            :src="getImageUrl(item.Id, 'Primary')"
-          />
+          <v-responsive aspect-ratio="1">
+            <v-avatar
+              color="card"
+              width="100%"
+              height="100%"
+              class="elevation-2"
+            >
+              <blurhash-image
+                v-if="item.ImageTags && item.ImageTags.Primary"
+                :item="item"
+              />
+              <v-icon v-else size="128" dark>mdi-account</v-icon>
```

Comment:
Why not do this fallback on BlurhashImage like all the items does?

BlurhashImage also has slots now, no need for ``v-else``

Reply:
Mostly because this happens:
![image](https://user-images.githubusercontent.com/19396809/113515648-10795a00-9576-11eb-9015-6e4b14e7ae33.png)

Reply:
576fdf1 fixes this by using slots properly in `blurhash-image` (There's no need to make a condition, you define the default content by putting it into the slot)

Reply:
I don't have a clue why this is happening. Maybe we can make the ``v-if`` [here](https://github.com/jellyfin/jellyfin-vue/blob/master/client/components/Layout/Images/BlurhashImage.vue#L26) an ``v-if-else``? I don't have a idea why this is happening, we're already getting artist images with proper icons in the home header.

I originally placed the condition so BlurhashImage is smart enough to provide icons for fallbacks but we're still able to pass our own custom slots in case we needed a different component or other needs. 

You *shouldn't* need those slots in any case, I'm testing your changes locally to see what could be going on.

Reply:
Okay, something strange iis going on. Master is showing me the ``mid-account`` icon for artists that don't have an image in the Home Header. Your PR shows the ``mdi-disc`` icon instead.

Artists that have images don't have any overlapping icon, so I think you should remove the passed slots altogether and let BlurhashImage handle it like we do in HomeHeader's master and artist cards.

Reply:
> I don't have a clue why this is happening. Maybe we can make the v-if here an v-if-else? I don't have a idea why this is happening, we're already getting artist images with proper icons in the home header.

That part of the code is completely gone.

The way you define a slot's default content isn't with v-if and v-else, but by putting the default inside of the slot.

Reply:
> You shouldn't need those slots in any case, I'm testing your changes locally to see what could be going on.

I removed the ones for the main artist and person images.

*However* one still remains in PersonList. In order to remove it, we should rework `getShapeFromItemType` to take `BaseItemDto | BaseItemPerson`. But that touches stuff outside of this PR's scope, so I'd rather do it in a followup.

Reply:
> Okay, something strange iis going on. Master is showing me the `mid-account` icon for artists that don't have an image in the Home Header. Your PR shows the `mdi-disc` icon instead.

It has always showed `mdi-disc` in the home header for me, since the home header lists albums, not persons.